test/tests/functional/pbs_hooksmoketest.py

```diff
@@ -463,6 +463,8 @@ def print_attribs(pbs_obj):
         # Character escaping '()' as the log_match is regexp
         msg_4 = msg_4.replace("(", "\(")
         msg_4 = msg_4.replace(")", "\)")
+        msg_4 = msg_4.replace("[", "\[")
+        msg_4 = msg_4.replace("]", "\]")
```

Comment:
What calling re.escape(msg_4)?  That does all the regex escaping you need.

Reply:
Good suggestion. Fixed.

Reply:
Actually `re.escape(msg_4)` does not work because `msg` need to remain a regular express. But `ev` needs to be escaped.